.github/workflows/release-docker.yml

```diff
@@ -34,16 +34,17 @@ jobs:
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
       - name: Set up rust toolchain
-        uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # tag=v1.0.7
+        uses: actions-rs/toolchain@88dc2356392166efad76775c878094f4e83ff746
         with:
           profile: minimal
-          toolchain: nightly
+          toolchain: nightly-2022-09-22
           target: wasm32-unknown-unknown
           default: true
           override: true
+          components: rustfmt, clippy
 
       - name: Cargo build
-        uses: actions-rs/cargo@844f36862e911db73fe0815f00a4a2602c279505 # tag=v1.0.3
+        uses: actions-rs/cargo@9e120dd99b0fbad1c065f686657e914e76bd7b72
```

Comment:
The `# tag=v1.0.3` or `# v1.0.3` is required, it will automatically be updated by Dependabot or Renovate

https://github.blog/changelog/2022-10-31-dependabot-now-updates-comments-in-github-actions-workflows-referencing-action-versions/

https://docs.renovatebot.com/modules/manager/github-actions/

> On another note, `# renovate: tag=v1.0.0`and `# tag=v1.0.0` have been deprecated, it is just `# v1.0.0` now for version pinning.